modules/ingest-grok/src/test/resources/rest-api-spec/test/ingest_grok/10_basic.yaml

```diff
@@ -0,0 +1,12 @@
+"Ingest grok installed":
+    - do:
+        cluster.state: {}
+
+    # Get master node id
+    - set: { master_node: master }
```

Comment:
ok so it's not that we need the master, but the id of the current master is the easiest one to find?